modules/nlu/src/backend/module-lifecycle/on-server-started.ts

```diff
@@ -122,11 +122,13 @@ export function getOnSeverStarted(state: NLUState) {
     }
 
     const { ducklingEnabled, ducklingURL, languageSources, modelCacheSize } = globalConfig
+    const parsedCachedSize = bytes(modelCacheSize)
+
     const parsedConfig: NLUEngine.Config = {
       languageSources,
       ducklingEnabled,
       ducklingURL,
-      modelCacheSize: bytes(modelCacheSize)
+      modelCacheSize: modelCacheSize && parsedCachedSize ? parsedCachedSize : Infinity
```

Comment:
you don't need to check for modelCacheSize to be truthy, if it's falsy, paarsedCachedSized wil be null.

Also you should check for parsedCachedSize to be greater than 0. this will prevent you from checking later on